policyengine_us/parameters/gov/states/ar/tax/income/rates/main/rate.yaml

```diff
@@ -1,55 +1,140 @@
 description: Arkansas taxes personal income at these amounts.
 metadata:
-  type: single_amount
+  type: single_amount 
   threshold_unit: currency-USD
   amount_unit: /1
   label: Arkansas income tax amount
+  period: year
   reference:
-  - title: Tax Legislation for 2022 Ark. Code § 26-51-201 (a)(1)(A)
```

Comment:
why remove the legal code reference? is there another we can include?

Reply:
The legal code reference has been re-added to the reference list.